config/settings/production.py

```diff
@@ -255,9 +257,24 @@
 
 HEROKU_TOKEN = env("HEROKU_TOKEN", default=None)
 HEROKU_APP_NAME = env("HEROKU_APP_NAME", default=None)
+
 if HEROKU_TOKEN and HEROKU_APP_NAME:
     METACI_WORKER_AUTOSCALER = "metaci.build.autoscaling.HerokuAutoscaler"
 
+# Autoscalers are defined per METACI_APP
+AUTOSCALERS = json.loads(env("AUTOSCALERS", default={}))
```

Comment:
```suggestion
AUTOSCALERS = json.loads(env("AUTOSCALERS", default="{}"))
```

The default for `env` needs to be a string so that it can be loaded as JSON.